### PR TITLE
New ppi 6/29/2020

### PR DESCRIPTION
**Pull request by Yevhenii-Bezeha**

### CODE REVIEW

**Comment by Yevhenii-Bezeha** [comment]
Updated load_stage for PPI.
Added new relationship_id and concept_class_id

**Review by hardhouse — DISMISSED** [review]

File: PPI/load_stage.sql [hunk 1]
```diff
@@ -22,8 +22,8 @@ DO $_$
 BEGIN
 	PERFORM VOCABULARY_PACK.SetLatestUpdate( 
 	pVocabularyName			=> 'PPI',
-	pVocabularyDate			=> TO_DATE ('2018-12-28' ,'yyyy-mm-dd'), -- Date of Version Update from PPI Codebook
-	pVocabularyVersion		=> 'Codebook Version 0.3.34',  -- Current Codebook Version from PPI Codebook
+	pVocabularyDate			=> TO_DATE ('2020-05-07' ,'yyyy-mm-dd'), -- Date of Version Update from PPI Codebook
+	pVocabularyVersion		=> 'Codebook Version 0.3.34 + COVID + MHWB',  -- Current Codebook Version from PPI Codebook
```
Comment (Alexdavv) [inline PPI/load_stage.sql:26]: @Yevhenii-Bezeha 
Is 'COVID + MHWB' something to be specified here or in references?
Isn't it another vocabulary?

File: working/manual_changes/2020/2020-06-29.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+--create new Relationship_id for PPI
+DO $_$
+BEGIN
+	PERFORM vocabulary_pack.AddNewRelationship(
+	pRelationship_name			=>'Has question source',
```
Comment (Alexdavv) [inline working/manual_changes/2020/2020-06-29.sql:5]: @Yevhenii-Bezeha 
"source of question" or "question's source"?

File: PPI/load_stage.sql (hunk 1 above)
Comment (Yevhenii-Bezeha) [reply PPI/load_stage.sql:26]: @Alexdavv 
We spoke with PPI team before about this. And now it is the best way to show currently PPI has not only concepts from the Codebook Version 0.3.34, but also from the surveys COVID and MHWB (mental health well being). COVID - it is a short name from PPI's new survey.

File: working/manual_changes/2020/2020-06-29.sql (hunk 2 above)
Comment (Yevhenii-Bezeha) [reply working/manual_changes/2020/2020-06-29.sql:5]: PPI team gave us the source. They call it - Appendix for each survey. And Question source - it it a name of one column there. So I used this relationship_id  as copy from that column, if I understood you correctly.

File: PPI/load_stage.sql (hunk 1 above)
Comment (Alexdavv) [reply PPI/load_stage.sql:26]: As far as I remember, the list of sources/references to the part of the terminology were never put to the vocabulary.vocabulary_version. Can you please check, BTW?

Why vocabulary_reference isn't working here?

File: working/manual_changes/2020/2020-06-29.sql (hunk 2 above)
Comment (Alexdavv) [reply working/manual_changes/2020/2020-06-29.sql:5]: Git it, but a little bit confusing. I'd prefer more explicit "source of question".
@dimshitc @mik-ohdsi What do you think?

File: PPI/load_stage.sql (hunk 1 above)
Comment (Yevhenii-Bezeha) [reply PPI/load_stage.sql:26]: I don't know, unfortunately. If you have some ideas about it maybe we could find the problem.
This vocabulary_reference is there for a long time.

**Pull request closed without merging**